### ISSUE TITLE
Create card view for API catalog

### ISSUE THREAD

**Issue by bajiat** [apinf/api-umbrella-dashboard#988]
# User story
> As an Apinf user I want to select between catalog views so that I can view either card catalog and table catalog.

# Wireframe
![Card catalog wireframe](https://raw.githubusercontent.com/apinf/docs/master/docs/Wireframes/Cards/1.%20Card1-%20Normal%20State.png)

Full list of wireframes can be found in our [documentation repository](https://github.com/apinf/docs/tree/master/docs/Wireframes/Cards).

# Definition of done
- [ ] Project includes card view design, implementing the card view, toggling implementation

**Comment by Nazarah** [comment]
There should be option to toggle between table and catelogue view in the catelogue menu

**Comment by Nazarah** [comment]
For the current homepage of APINF, the cards for latest/most recent catelogues should change as per new design, so that it is consistence with the new card view in catelogue menu.

**Comment by Nazarah** [comment]
I will work on the wireframes.

**Comment by brylie** [comment]
@Nazarah and @elnzv the task description now contains one wireframe and a link to the card catalog wireframes folder in our documentation repository.

**Comment by bajiat** [comment]
@nnn What's the status of the task? Are you waiting on container task or are you able to continue even if it is work in progress?
